docs/development/guides/07_working_with_datasets.ipynb

```diff
@@ -64,28 +64,6 @@
     "])"
    ]
   },
-  {
```

Comment:
This ray visualization isn't playing well in the CI. Haven't been able to reproduce in a local container. It's not critical, so I'm removing it for now